invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -8,13 +8,28 @@
 
 """Names datastreams, transformers, writers and readers."""
 
+import requests
 from invenio_access.permissions import system_identity
 from invenio_records.dictutils import dict_lookup
 
 from ...datastreams.errors import TransformerError
+from ...datastreams.readers import SimpleHTTPReader
 from ...datastreams.transformers import XMLTransformer
 
 
+class OrcidHTTPReader(SimpleHTTPReader):
+    """ORCiD HTTP Reader."""
+
+    def __init__(self, *args, test_mode=True, **kwargs):
```

Comment:
I still kept this to enforce the URL, otherwise we just use the `SimpleHTTPReader`.